articles/storage/common/azure-defender-storage-configure.md

```diff
@@ -93,21 +93,21 @@ Use an Azure Policy to enable Microsoft Defender for Cloud across storage accoun
 
 ### [PowerShell](#tab/azure-powershell)
 
-To enable Microsoft Defender for Storage for a storage account via PowerShell, first make sure you have installed the [Az.Security](https://www.powershellgallery.com/packages/Az.Security) module. Next, call the [Enable-AzSecurityAdvancedThreatProtection](/powershell/module/az.security/enable-azsecurityadvancedthreatprotection) command. Remember to replace values in angle brackets with your own values:
+To enable Microsoft Defender for Storage for a storage account via PowerShell, first make sure you have installed the [Az.Security](https://www.powershellgallery.com/packages/Az.Security) module. Next, call the [Enable-AzSecurityAdvancedThreatProtection](https://github.com/Azure/azure-powershell/blob/main/src/Security/Security/help/Enable-AzSecurityAdvancedThreatProtection.md) command. Remember to replace values in angle brackets with your own values:
 
 ```azurepowershell
 Enable-AzSecurityAdvancedThreatProtection -ResourceId "/subscriptions/<subscription-id>/resourceGroups/<resource-group>/providers/Microsoft.Storage/storageAccounts/<storage-account>/"
 ```
 
-To check the Microsoft Defender for Storage setting for a storage account via PowerShell, call the [Get-AzSecurityAdvancedThreatProtection](/powershell/module/az.security/get-azsecurityadvancedthreatprotection) command. Remember to replace values in angle brackets with your own values:
+To check the Microsoft Defender for Storage setting for a storage account via PowerShell, call the [Get-AzSecurityAdvancedThreatProtection](https://github.com/Azure/azure-powershell/blob/main/src/Security/Security/help/Get-AzSecurityAdvancedThreatProtection.md) command. Remember to replace values in angle brackets with your own values:
```

Comment:
We need to use relative links, and also we don't want to link to GitHub here. The original link is correct. Please revert.

Reply:
Thanks for the info and sorry for the circumstances. I was not aware that the these links are relative to the microsoft website and not github. If you read the article on github, you still get broken links.

Reply:
@Patchfox - No problem. The articles are designed to be consumed on the Microsoft Docs site, not on GitHub. While you can read them on GitHub, the links shouldn't be expected to work.